File: backend/internconnectbackendspring/src/main/java/com/internconnect/internconnectbackendspring/model/Company.java
package com.internconnect.internconnectbackendspring.model;

import jakarta.persistence.*;

import java.io.Serializable;

@Entity
@Table(name = "company")
public class Company implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(name = "email", nullable = false, length = 255)
    private String email;
    @Column(name = "pass", nullable = false, length = 255)
    private String pass;
    @Column(name = "companyName", nullable = false, length = 255)
    private String companyName;
    @Column(name = "industry", nullable = false, length = 255)
    private String industry;
    @Column(name = "contactPerson", nullable = false, length = 255)
    private String contactPerson;
    @Column(name = "contactPhone", nullable = false, length = 255)
    private String contactPhone;
}

File: frontend/internconnectfrontendclient/composeApp/src/commonMain/kotlin/com/lazar/internconnectfrontendclient/data/model/Company.kt
package com.lazar.internconnectfrontendclient.data.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "company")
data class Company(
    @PrimaryKey(autoGenerate = true)
    val id: Long,
    @ColumnInfo(name = "email")
    val email: String,
    @ColumnInfo(name = "pass")
    val pass: String,
    @ColumnInfo(name = "companyName")
    val companyName: String,
    @ColumnInfo(name = "industry")
    val industry: String,
    @ColumnInfo(name = "contactPerson")
    val contactPerson: String,
    @ColumnInfo(name = "contactPhone")
    val contactPhone: String,
)